apps/storefront/src/shared/service/request/fetch.ts

```diff
@@ -1,51 +1,25 @@
-import { snackbar } from '@/utils';
-
-import { RequestType } from './base';
-
 const originFetch = window.fetch;
 
-const responseResult = (path: string, res: any, resolve: any, init: any) => {
-  if (path.includes('current.jwt')) return res.text();
-
+const responseResult = (res: any, resolve: any, init: any) => {
   if (init.method === 'DELETE') {
     resolve();
   }
```

Comment:
This `init.method === 'DELETE'` is a bit weird, it'll mark any failed request as a success for the DELETE method.

I don't think that's desired, as it'll masquerade any errors while deleting.